themes/bootstrap3/templates/myresearch/menu.phtml

```diff
@@ -80,22 +80,26 @@
   <h3><?=$this->transEsc('Your Lists')?></h3>
   <div class="myresearch-menu">
     <a href="<?=$this->url('myresearch-favorites')?>"<?=$this->active == 'favorites' ? ' class="active"' : ''?>>
-      <i class="fa fa-fw fa-star" aria-hidden="true"></i> <?=$this->transEsc('Your Favorites')?>
+      <?=$this->icon('user-favorites') ?><?=$this->transEsc('Your Favorites')?>
     </a>
     <?php $lists = $user->getLists() ?>
     <?php
-      // Use a variable so that we can output this nicely without whitespace that would get underlined:
-      $publicInd = '<i class="fa fa-globe" title="' . $this->transEscAttr('public_list_indicator') . '">'
-        . '</i> <span class="sr-only">(' . $this->transEsc('public_list_indicator') . ')</span>';
```

Comment:
The sr-only text seems to have gotten lost in the refactoring here, which is causing a test to fail.